docs/user-guide/transformer.md

```diff
@@ -85,76 +85,14 @@ transformerConfig:
       features: # list of features to be retrieved
         - name: merchant_t1_discovery:t1_estimate # feature name. make sure to include feature set
           valueType: DOUBLE # feature value type
-          defaultValue: '0' # default value if feature value not exist
+          defaultValue: "0" # default value if feature value not exist
 ```
 
 > To learn more about JSON Path syntax, please go to [this link](https://goessner.net/articles/JsonPath/), [this](https://restfulapi.net/json-jsonpath/) or [this](https://support.smartbear.com/alertsite/docs/monitors/api/endpoint/jsonpath.html).
 
 ### List of Supported UDFs
```

Comment:
It's not UDF. UDF is User-Defined Function. We should replace the mention of UDF in our document with "expression"